frontend/src/components/Chat.js

```diff
@@ -14,14 +13,21 @@ import {
   Typography,
 } from '@mui/material'
 import SendIcon from '@mui/icons-material/Send'
+import { collabSocket } from '../utils/socket'
+import { getCollabRoomId } from '../utils/main'
+import { UserContext } from '../contexts/UserContext'
 
 const Chat = ({ room }) => {
   collabSocket.on('receive-message', (msg) => {
     setChatMessages([...chatMessages, msg])
   })
+  collabSocket.on('restore-chat', (chat) => {
+    setChatMessages(chat.map((msg) => msg.content))
```

Comment:
Note: right now we don't show usernames in the chat box. But eventually, if we wanna show some UI indicator to differentiate between msgs sent by user A or user B, we can make use of `msg.from` to see who sent the message